network-test-app/app.js

```diff
@@ -0,0 +1,57 @@
+const { Client } = require("pg");
+const express = require("express");
+const app = express()
+
+const port = process.env.PORT;
+
+app.get("/", async (request, response) => {
+    pgServiceCredentials = process.env.DATABASES_FOR_POSTGRESQL_CONNECTION
+    if(pgServiceCredentials != "" && pgServiceCredentials != undefined){
```

Comment:
This app won't write a response at all if there is no postgres configured. Also, you might be able to use a truthy check here to simplify the condition.

```
if (!!pgServiceCredentials) {
```